src/prx/main.py

```diff
@@ -221,11 +224,21 @@ def check_assumptions(
     )
 
 
+def parse_rinex_nav_or_obs_file(rinex_file_path: Path):
```

Comment:
This convenience function was actually only used to warm up the parser cache, so moved it here.